external/malmo/Minecraft/run/options.txt

```diff
@@ -1,102 +0,0 @@
-version:922
```

Comment:
@JiangfengLi Please checkout the version of this file from the master branch with:

    git checkout master -- external/malmo/Minecraft/run/options.txt